etl/generative_model_pipelines/open_ai_generative_model_pipeline.py

```diff
@@ -59,17 +66,15 @@ def enrich_record(self, record: Record) -> Record:
         Return the original Record if OpenAiResourceParams.enrichment_type is not a field of Record.
         """
 
-        if self.openai_resource_params.enrichment_type in record.model_fields.keys():
-            match self.openai_resource_params.record_type:
+        if self.__openai_pipeline_config.enrichment_type in record.model_fields:
```

Comment:
This is a big indent with no else case. Just do `not in` `return record` here?

Reply:
Yes. this is better. I have changed the if statement.